src/aura/HH_Container/HH_ContainerHelper.js

```diff
@@ -106,7 +106,9 @@
 
             // tell our visualforce page we are done loading
             var event = $A.get("e.c:HH_ContainerLoadedEvent");
-            event.fire();
```

Comment:
This isn't exactly dchasman's pattern.  Doug was setting up the actions in the container app (in this case actions you would have defined in HH_ContainerController.js) to handle events fired (via attribute shortcut to addEventHandler in hosting visualforce page).  Those controller actions would delegate to the external methods passed in to the container app from the visualforce page.

Here, you're passing in the methods but foregoing the addEventHandler step altogether.  In which case, your methods aren't actually event handlers, they're just methods you're getting a reference to and then calling.  So, the whole $A.get('e.c:HH_ContainerLoadedEvent') etc is superfluous.  Just call the method, and pass in an argument if you need to pass something in.

Not sure if your more direct way has any other implications.